docker-machine-nfs.sh

```diff
@@ -398,7 +424,8 @@ configureBoot2Docker()
   # (this will override an existing /var/lib/boot2docker/bootlocal.sh)
 
   local bootlocalsh='#!/bin/sh
-  sudo umount /Users'
+  sudo umount /Users
+  sudo umount /c/Users'
```

Comment:
Won't this cause warnings if the volume does not exist? Maybe better to create a bootlocal depending on the OS?